### PR TITLE
Added ownerside participant, added GetContract action, expanded contr…

### PR DESCRIPTION
**Pull request by DmitryKleshchov**
- Added `Ownerside` participant type to `Add Participant` action
- Added `Get Contract Details` action
- Exposed more properties for contract model
- Fixed a bug where `Add Participant` would create duplicate parties sometimes
- Fixed a bug with `Delete Contract` action not having a correct return type specified for `200` response. 

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Oneflow/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1223,117 +1261,101 @@
...
             "start_date": {
               "type": "string",
-              "description": "start_date",
-              "x-ms-summary": "Contract Start Date"
+              "x-ms-summary": "Contract start date"
             },
             "type": {
               "type": "string",
-              "description": "type",
+              "description": "no_duration/single_period/recurring/recurring_two_periods",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Oneflow/apiDefinition.swagger.json:1293]: @DmitryKleshchov please make more detailed descriptions if you want connector users to choose between possible values. You can refer to the [guidelines for creating strings](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) or [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).

Comment (v-EgorKozhadei) [inline certified-connectors/Oneflow/apiDefinition.swagger.json:1293]: @DmitryKleshchov please make more detailed descriptions. You can refer to the [guidelines for creating strings](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) or [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).

File: certified-connectors/Oneflow/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1223,117 +1261,101 @@
...
               "type": "boolean",
-              "description": "is_recurring",
-              "x-ms-summary": "Is Contract Recurring"
+              "x-ms-summary": "Is contract recurring"
             },
             "lifecycle_state": {
               "type": "string",
-              "description": "lifecycle_state",
-              "x-ms-summary": "Lifecycle State"
+              "description": "awaiting/active/ended",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Oneflow/apiDefinition.swagger.json:1332]: @DmitryKleshchov please make more detailed descriptions if you want connector users to choose between possible values. You can refer to the [guidelines for creating strings](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) or [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).

Comment (v-EgorKozhadei) [inline certified-connectors/Oneflow/apiDefinition.swagger.json:1332]: @DmitryKleshchov please make more detailed descriptions if you want connector users to choose between possible values. You can refer to the [guidelines for creating strings](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) or [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).

File: certified-connectors/Oneflow/apiProperties.json [hunk 3]
```diff
@@ -18,8 +18,8 @@
         "type": "string",
         "uiDefinition": {
           "displayName": "Email address",
-          "description": "Oneflow user email address",
-          "tooltip": "Provide Oneflow user email address",
+          "description": "Login of Oneflow user the steps will be executed under",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Oneflow/apiProperties.json:21]: @DmitryKleshchov please make sure everything is grammatically correct and is free of typographical errors. You can refer to the [guidelines for creating strings](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Comment (v-EgorKozhadei) [inline certified-connectors/Oneflow/apiProperties.json:21]: @DmitryKleshchov please make sure everything is grammatically correct and is free of typographical errors. You can refer to the [guidelines for creating strings](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/Oneflow/apiProperties.json [hunk 4]
```diff
@@ -57,7 +57,6 @@
         }
       }
     ],
-    "publisher": "Oneflow AB",
-    "stackOwner": "Oneflow AB"
+    "publisher": "Dmitry Kleshchov"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Oneflow/apiProperties.json:60]: @DmitryKleshchov can you please explain why did you remove the `Oneflow AB` from the [publisher and stackOwner properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)?

Comment (v-EgorKozhadei) [inline certified-connectors/Oneflow/apiProperties.json:60]: @DmitryKleshchov can you please explain why did you remove the `Oneflow AB` from the [publisher and stackOwner properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)?

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @DmitryKleshchov! Please check comments after the review, thank you!

File: certified-connectors/Oneflow/apiDefinition.swagger.json (hunk 1 above)
Comment (DmitryKleshchov) [reply certified-connectors/Oneflow/apiDefinition.swagger.json:1293]: Hi @v-EgorKozhadei  ! 
Descriptions like above are used only in output models.
 If we make them too long, they won't fit in the output window of the flow constructor, thus becoming harder to understand. As you guessed, these are just the output values of enum. 
The names of parameters are self-explanatory, so we don't need to add more description below, the only option is either to remove description completely, or to have these technical values inside to assist end users in understanding the data that can be contained in these fields: 
image

File: certified-connectors/Oneflow/apiProperties.json (hunk 4 above)
Comment (DmitryKleshchov) [reply certified-connectors/Oneflow/apiProperties.json:60]: @v-EgorKozhadei  I did not, in fact, this looks like an old version of apiProperties.json. I'll need to check it out, thanks for pointing this out.

File: certified-connectors/Oneflow/apiDefinition.swagger.json (hunk 1 above)
Comment (v-EgorKozhadei) [reply certified-connectors/Oneflow/apiDefinition.swagger.json:1293]: @DmitryKleshchov thanks for the answer! I think we can keep it as it is.

File: certified-connectors/Oneflow/apiProperties.json (hunk 3 above)
Comment (DmitryKleshchov) [reply certified-connectors/Oneflow/apiProperties.json]: @v-EgorKozhadei should be fixed now

File: certified-connectors/Oneflow/apiProperties.json (hunk 4 above)
Comment (DmitryKleshchov) [reply certified-connectors/Oneflow/apiProperties.json]: @v-EgorKozhadei should be fixed now

**Review by sriyen-msft — APPROVED** [review]

**Review by sriyen-msft — APPROVED** [review]
